kong/cmd/utils/ssl.lua

```diff
@@ -0,0 +1,71 @@
+local utils = require "kong.tools.utils"
+local pl_path = require "pl.path"
+local pl_utils = require "pl.utils"
+local pl_dir = require "pl.dir"
+local log = require "kong.cmd.utils.log"
+local fmt = string.format
+
+local _M = {}
+
+local SSL_FOLDER = "ssl"
+local SSL_CERT = "kong-default.crt"
+local SSL_CERT_KEY = "kong-default.key"
+local SSL_CERT_CSR = "kong-default.csr"
+
+function _M.get_ssl_cert_and_key(kong_config, nginx_prefix)
+  local ssl_cert, ssl_cert_key
+  if kong_config.ssl_cert and kong_config.ssl_cert_key then
+    ssl_cert = kong_config.ssl_cert
+    ssl_cert_key = kong_config.ssl_cert_key
+  else
+    ssl_cert = pl_path.join(nginx_prefix, SSL_FOLDER, SSL_CERT)
+    ssl_cert_key = pl_path.join(nginx_prefix, SSL_FOLDER, SSL_CERT_KEY)
+  end
+
+  -- Check that the files exist
+  if not pl_path.exists(ssl_cert) then
+    return false, "Can't find SSL certificate at: "..ssl_cert
```

Comment:
also: to stay consistent, don't capitalize error messages. In the CLI, the errors always appear as:

```lua
Error: can't find SSL certificate at: ...
```